packages/firebase_app_installations/firebase_app_installations/pubspec.yaml

```diff
@@ -2,7 +2,7 @@
 name: firebase_app_installations
 description: A Flutter plugin allowing you to use Firebase Installations.
 version: 0.1.0+13
-homepage: https://firebase.flutter.dev/docs/installations/overview
+homepage: https://firebase.google.com/docs/projects/manage-installations#flutter
```

Comment:
@kevinthecheung - this seems fine to me, but curious about your thoughts on whether this is the correct place for the homepage?

Reply:
Yeah, that's where all of the related info is. (We don't treat it as its own product.)